.eslintrc

```diff
@@ -0,0 +1,33 @@
+{
+  "root": true,
+  "extends": "airbnb-base",
+  "env": {
+    "node": true,
+    "es6": true,
+    "mocha": true,
+    "browser": true
+  },
+  "rules": {
+    "one-var": 0,
+    "one-var-declaration-per-line": 0,
+    "new-cap": 0,
+    "consistent-return": 0,
+    "no-param-reassign": 0,
+    "import/no-unresolved": 0,
+    "comma-dangle": 0,
+    "curly": ["error", "multi-line"],
+    "no-shadow": ["error", { "allow": ["req", "done", "res", "err"] }],
+    "valid-jsdoc": [1, {
+      "requireReturn": false,
+      "requireReturnType": false,
+      "requireParamDescription": false,
+      "requireReturnDescription": false
```

Comment:
Nice work @prieste.
But do you have reasons for turning the linting rules for `valid-jsdoc` off? Ideally they should be on.

Reply:
I usually don't use jsDocs, old habit, I'll turn it back on